runtime.md

```diff
@@ -1,5 +1,51 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information is stored in `/run/oci`.
+The directory structure for a container is `<root>/<containerID>/state.json`.
+
+* **id** (string) ID is the container's ID.
```

Comment:
On Wed, Aug 26, 2015 at 10:12:52AM -0700, Brandon Philips wrote:
> > +* **id** (string) ID is the container's ID.
> 
> We need to add a comment that the reason we have an ID is because
> the hooks only get an open fd to this json document so they won't
> know the unique ID which may be useful for removal, etc.

I wasn't clear on how it would be useful for removal.  Everyone got on
board when someone mentioned network teardown, but the connection
between that and the container ID wasn't clear to me.  Not that we
have to be completely specific when motivating this entry.  I'm just
pointing out that if we want to explain why we need the container ID
in a post-stop hook, we might want to provide more detail than we had
in today's meeting.